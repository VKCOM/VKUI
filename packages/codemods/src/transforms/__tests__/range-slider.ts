jest.autoMockOff();

import { defineTest } from 'jscodeshift/dist/testUtils';

const name = 'range-slider';
const fixtures = ['basic', 'renamed', 'imports'] as const;

describe(name, () => {
  fixtures.forEach((test) =>
    defineTest(__dirname, name, global.TRANSFORM_OPTIONS, `${name}/${test}`, {
      parser: 'tsx',
    }),
  );
});
