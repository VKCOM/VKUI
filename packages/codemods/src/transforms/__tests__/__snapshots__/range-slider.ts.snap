// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`range-slider transforms correctly 1`] = `
"import { Slider } from '@vkontakte/vkui';
import React from 'react';
import '@vkontakte/vkui/dist/vkui.css';

const App = () => {
  const props = { id: 'RangeSlider' };
  return (
    (<React.Fragment>
      <Slider multiple />
      <Slider multiple {...props} boolValue />
      <Slider multiple test="123" />
      <Slider multiple test="123">inner</Slider>
    </React.Fragment>)
  );
};"
`;

exports[`range-slider transforms correctly 2`] = `
"import { Slider, Slider as VKUIRangeSlider } from '@vkontakte/vkui';
import React from 'react';
import '@vkontakte/vkui/dist/vkui.css';

const App = () => {
  const props = { id: 'RangeSlider' };
  return (
    (<React.Fragment>
      <VKUIRangeSlider multiple {...props} boolValue />
      <Slider test="123" />
    </React.Fragment>)
  );
};"
`;

exports[`range-slider transforms correctly 3`] = `
"import { Slider as VKUIRangeSlider } from '@vkontakte/vkui';
import React from 'react';
import '@vkontakte/vkui/dist/vkui.css';

const App = () => {
  const props = { id: 'RangeSlider' };
  return (
    (<React.Fragment>
      <VKUIRangeSlider multiple {...props} boolValue />
    </React.Fragment>)
  );
};"
`;
